### PR TITLE
Dynamic stats

### PR DESCRIPTION
**Pull request by lyzidiamond**
Updates stats dynamically. This is how it works:
- When you click a feature, that feature is passed as an argument into the `update_stats()` function.
- `update_stats()` iterates over each amenity/stat in the list and does the following:
  - Skips `GEOID10` and `tract_name`
  - Checks to see if there are parks in the tract. If yes, sets value of `tract_amenity` to actual tract value. If no, sets value of tract_amenity to "N/A".
  - In line 73: Selects DOM element by amenity ID (the IDs match the amenity names), updates the HTML to include the stat value.

This PR also removes stats from popups.

### CODE REVIEW

File: js/parks_map.js [hunk 1]
```diff
@@ -44,23 +44,37 @@ function style(feature) {
 function popup(feature) {
```
Comment (livienyin) [inline js/parks_map.js:44]: 1). Not sure where to add this comment- would it be easy to only display up to tenths decimal place for Total Park Acreage?

2). Tract 38.02 might have bad data? Southern part of tract area is clickable but northern area is not. It says there is 1 park but "Biggest Park" is blank.

**Comment by livienyin** [comment]
Thanks for adding the explanation comment up top! After merging, I will remove the fake default values in the HTML.

**Comment by lyzidiamond** [comment]
1. YES you read my mind, that was the next thing I was gonna do.
2. I think it has to do with tile loading. If you pan the map a ways, you can click anywhere on the polygon. Another argument for taking out zooming?

**Comment by livienyin** [comment]
1. Looks like certain tracts, for whatever reason, aren't rounding total acreage- like tract 38.03 or 36. Finicky little tracts! NBD if it's time consuming to figure out why some aren't rounding, more of a product polishing fix. Can add it as an issue. Displaying averages in the table would be a higher priority. Up to you!
2. We can definitely revisit this- I think it's still nice to have the option to click on zoom in/out since we haven't built in "search by address" so users may need to zoom in close enough to see the exact boundaries of a tract area. I assume people generally aren't familiar with their census tract area #.

**Comment by lyzidiamond** [comment]
I looked through the park acres, and I don't see any really long, not-rounded numbers... I'm gonna merge the PR and let's see if the problem persists.

**Comment by livienyin** [comment]
Okay sounds good!

**Pull request merged**